### ISSUE TITLE
generating a new simulated device with different name

### ISSUE THREAD

**Issue by PhilLAL** [tango-controls/SimulatorDS#9]
Hi Sergi,
while testing SimulatorDS, I captured a device which I would like to replicate into several devices.
I succeed it by editing the PCK file and changing the name X/Y/Z (line 9 into my file).
Is it the good way to do this or is there a smarter way?

Regards.

**Comment by sergirubio** [comment]
I'm adding a "suffix" option to differentiate original devices from new ones (e.g., just adding "-sim" at the end).

Would it be enough or you need something smarter? 

I could enable a regexp replacement chain like "regexp1:replace1;regexp2:replace2;regexp3:replace3"

**Comment by PhilLAL** [comment]
Thank you for this proposal.

Since we have a way to change the name of the device, I think this is not a priority, so the suffix will be perfect for me.
In our context, we will use a simulation TANGO server on which we configured the SimlulatorDS with as much simulated DS as we can, so we use the same device names as in reality but we would "capture" only once each DS, since the behaviour is the same for each device.

**Comment by PhilLAL** [comment]
I tested again this day, and the new suffix is useful :)
